openrtb_ext/convert_down.go

```diff
@@ -174,9 +176,9 @@ func moveRewardedFrom26ToPrebidExt(i *ImpWrapper) error {
 	return nil
 }
 
-// clear26Fields sets all fields introduced in OpenRTB 2.6 to default values, which
+// Clear26Fields sets all fields introduced in OpenRTB 2.6 to default values, which
 // will cause them to be omitted during json marshal.
-func clear26Fields(r *RequestWrapper) {
+func Clear26Fields(r *RequestWrapper) {
```

Comment:
Helper functions are now exported for adapters which need to clear new fields for compatibility.